src/main/java/me/zero/alpine/listener/Listener.java

```diff
@@ -87,13 +88,10 @@ public int getPriority() {
      */
     @Override
     public void invoke(T event) {
-        if (filters.length > 0) {
-            for (Predicate<T> filter : filters) {
-                if (!filter.test(event)) {
-                    return;
-                }
-            }
+        if (Arrays.stream(filters).anyMatch(pred -> !pred.test(event))) {
```

Comment:
Streaming the filters and creating an impure lambda should vastly slow the performance, I don't think the JIT is saving us this time. Given that the intent of this function is to be performant, this should be reverted.

Reply:
oh i didnt realize this, okay change is reverted :+1: